### ISSUE TITLE
Get verification code by phone call

### ISSUE THREAD

**Issue by abbas-oveissi** [grishka/Houseclub#239]
**Is your feature request related to a problem? Please describe.**
In Iran, I tried multiple times but I couldn't get the verification code by SMS. 

**Describe the solution you'd like**
Adding "phone call" option will be resolved this issue.

**Describe alternatives you've considered**

**Additional context**

**Comment by ahmed-nossier** [comment]
your account not suspended ?!

**Comment by abbas-oveissi** [comment]
No, of course I haven't used the App a lot. I only used it to test my pull request.

**Comment by caasn** [comment]
Search for "receive sms online" and try getting in with a trash sms number.

**Comment by abbas-oveissi** [comment]
@caasn 
There is no need to use SMS online services, I've implemented "Phone call" functionality on my pull request ( #237 ). If @grishka agrees with it, it can be merged.

**Comment by caasn** [comment]
Clubhouse doesn't offer signing up via phone calls as far as I know.

**Comment by abbas-oveissi** [comment]
@caasn 
You are wrong. ClubHouse API has an endpoint (call_phone_number_auth) for this. Check my code:

https://github.com/grishka/Houseclub/blob/58591df0632c765962840c7ea077ae1404ae5ff6/Houseclub/src/main/java/me/grishka/houseclub/api/methods/CallPhoneNumberAuth.java

**Comment by caasn** [comment]
That's Houseclub, I was talking about Clubhouse. I just haven't heard about singing up in another way than sms.

**Comment by abbas-oveissi** [comment]
@caasn 
The Houseclub uses Clubhouse's API, so when you see any functionality on Houseclub, you should sure that Clubhouse has this functionality too.

In the signup flow on Clubhouse, If you click multiple times on "didn't get it? tap to resend", sometimes you can use phone call option.

**Comment by caasn** [comment]
Ah, okay.

**Comment by Smscodehub** [comment]
The `call_phone_number_auth` endpoint you implemented in PR #237 is correct for enabling phone call verification via Clubhouse's API. If SMS delivery remains unreliable, using a virtual number service can bypass regional restrictions. For consistent access to verification codes, consider [getsmsonline.com](https://getsmsonline.com) for professional disposable numbers.
